spread.yaml

```diff
@@ -42,6 +42,7 @@ environment:
     NO_PROXY: "127.0.0.1"
     NEW_CORE_CHANNEL: "$(HOST: echo $SPREAD_NEW_CORE_CHANNEL)"
     SRU_VALIDATION: "$(HOST: echo ${SPREAD_SRU_VALIDATION:-0})"
+    PPA_VALIDATION: "$(HOST: echo ${SPREAD_PPA_VALIDATION:-})"
```

Comment:
SRU_VALIDATION defaults to 0 and seems to be a boolean. PPA_VALIDATION is named alike but defaults to an empty string and seems to be used as a PPA name. I think this warrants a different name. Perhaps `PPA_VALIDATION_NAME`?

In addition, can you please add a comment about the purpose of the variable here (in the yaml).

Reply:
Done, thanks for reviewing!!!